SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/TypeStructure.cpp

```diff
@@ -456,6 +467,14 @@ FUnrealFlatRepData GetFlatRepData(TSharedPtr<FUnrealType> TypeInfo)
 			case COND_OwnerOnly:
 				Group = REP_SingleClient;
 				break;
+			case COND_InitialOnly:
+				Group = REP_InitialOnly;
+				break;
+			case COND_InitialOrOwner:
+				UE_LOG(LogSpatialGDKSchemaGenerator, Error,
+					   TEXT("COND_InitialOrOwner not supported. COND_None will be used instead. %s::%s"), *TypeInfo->Type->GetName(),
```

Comment:
You mean this will fall through the same as COND_None? Is it worth mentioning that Group will be set to REP_MultiClient? Maybe its worth considering re-writing this with all cases covered and a default case, rather than a fallback to MultiClient for clarity.

Reply:
Yes, COND_None will be used as the replication condition for this property. This message is intended for the GDK Unreal user, which won't be aware of REP_*, only COND_*.